src/tagging-compliance/resource_parsing.py

```diff
@@ -105,7 +105,10 @@ def lambda_handler(event, context):
                     subset_list.append(subset_value)
         # Push resource list to Prometheus metric
         for resource in subset_list:
-            resource_list_guage.labels(resource, lambda_item["Region"], account_id).set(
+            region_id = lambda_item["Region"]
+            region_name = region_names.get(region_id, region_id)
```

Comment:
have you tested this lambda function?

Reply:
Updated in 3838411bde6274f9fc8b0be0737617065374b737

Reply:
not in this file. Editing was done in another resource list file. This has been reverted to original state